test/data/copt_covdepsurf_example.yaml

```diff
@@ -0,0 +1,439 @@
+generator: cti2yaml
+cantera-version: 2.6.0a2
+date: Wed, 04 Aug 2021 14:46:21 -0400
+input-files: [blondal_mech.cti]
+
+units: {length: cm, quantity: mol, activation-energy: kcal/mol}
+
+phases:
+- name: gas
+  thermo: ideal-gas
+  elements: [C, O, N]
+  species: [O2, CO2, CO, N2]
+  transport: mixture-averaged
+  state:
+    T: 300.0
+    P: 1.01325e+05
+- name: coverages_0
+  thermo: coverage-dependent-surface
+  elements: [C, O, Pt]
+  species: [Pt, OC_Pt, CO2_Pt, C_Pt, O_Pt]
+  kinetics: surface
+  reactions: [reactions]
+  state:
+    T: 500.0
+    P: 1.01325e+05
+    coverages: {Pt: 0.5, OC_Pt: 0.5, CO2_Pt: 0.0, C_Pt: 0.0, O_Pt: 0.0}
+  site-density: 2.72e-09
+- name: coverages_1
+  thermo: coverage-dependent-surface
+  elements: [C, O, Pt]
+  species: [Pt, OC_Pt, CO2_Pt, C_Pt, O_Pt]
+  kinetics: surface
+  reactions: [reactions]
+  state:
+    T: 625.0
+    P: 1.01325e+05
+    coverages: {Pt: 0.20, OC_Pt: 0.13, CO2_Pt: 0.21, C_Pt: 0.44, O_Pt: 0.02}
+  site-density: 2.72e-09
+  reference-state-coverage: 0.22
```

Comment:
Are all these phase definitions identical except for the state, site density and reference state coverage? If that's the case, I think it would be better to run the tests by just defining one `SurfPhase` object and changing it's state and checking the thermo properties against the reference values.